src/effects/RealtimeEffectList.cpp

```diff
@@ -8,6 +8,7 @@
 
 #include "RealtimeEffectList.h"
 #include "RealtimeEffectState.h"
+#include "RealtimeEffectStateUI.h"
```

Comment:
The `#include` that made the bad dependency cycle is still here, but not needed to build now so easily deleted.